resources/config-defaults.edn

```diff
@@ -147,6 +147,9 @@
  ;; enable /api/permissions/:user
  :enable-permissions-api false
 
+ ;; Should the catalogue be public and visible to everyone, i.e. also to not logged-in users.
+ :catalogue-is-public true
```

Comment:
I thought we agreed that the default would be `false`. When did this change? If we use `true` as the default it needs to be prominently stated in the changelog, at least.

Reply:
I changed them all to false now and we'll take it to use case by case.